### PR TITLE
Document about building from source

### PR DESCRIPTION
**Pull request by dkalinowski**

### CODE REVIEW

File: docs/build_from_source.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Building from source
+
+This document gives information how to build docker images from source with different variants.
```
Comment (dtrawins) [inline docs/build_from_source.md:3]:
```suggestion
This document gives information how to build docker images and the binary package from source with different variants.
```

File: docs/build_from_source.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+## Prerequisites
+
+1. [Docker Engine](https://docs.docker.com/engine/)
+1. Ubuntu 20.04 or RedHat 8.7 host
+1. make
+1. bash
+
```
Comment (atobiszei) [inline docs/build_from_source.md:11]: 1., 1., 1., 1.?

Comment (dkalinowski) [reply docs/build_from_source.md:11]: This is intended, md format is able to convert it into 1,2,3,4.

Comment (dkalinowski) [reply docs/build_from_source.md:11]: View generated md file

File: docs/build_from_source.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+     ├── intel-opencl-icd-dbgsym_<version>_amd64.deb
+     ├── intel-opencl-icd_<version>_amd64.deb
+     ├── libigdgmm12_<version>_amd64.deb
+     └── libigdgmm12_<version>_amd64.deb
+```
+and run make docker_build with parameter: INSTALL_DRIVER_VERSION=dg2.
+
+Example:
+```
+make docker_build BASE_OS=ubuntu INSTALL_DRIVER_VERSION=dg2
```
Comment (dtrawins) [inline docs/build_from_source.md:71]: base os was not explained

File: docs/build_from_source.md (hunk 2 above)
Comment (dkalinowski) [reply docs/build_from_source.md:11]: https://github.com/openvinotoolkit/model_server/blob/build-from-src/docs/build_from_source.md

File: docs/build_from_source.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+├── ovms.tar.gz.sha256
+├── ovms.tar.xz
+├── ovms.tar.xz.metadata.json
+├── ovms.tar.xz.sha256
+└── thirdparty-licenses
+```
+
+## Building Options
+
+### `INSTALL_DRIVER_VERSION`
```
Comment (dtrawins) [inline docs/build_from_source.md:49]: old driver version can be installed without downloading first

File: docs/build_from_source.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+Example:
+```bash
+make docker_build NVIDIA=1 OV_USE_BINARY=0 OV_SOURCE_BRANCH=releases/2022/3 OV_CONTRIB_BRANCH=releases/2022/3
+```
+```bash
+docker run -it --gpus all -p 9178:9178 -v ${PWD}/models/public/resnet-50-tf:/opt/model openvino/model_server:latest-cuda --model_path /opt/model --model_name resnet --target_device NVIDIA
+```
+
+Read more detailed usage in [developer guide](b/develop/docs/developer_guide.md).
```
Comment (dtrawins) [inline docs/build_from_source.md:97]: link?

Comment (dkalinowski) [reply docs/build_from_source.md]: fixed

File: docs/build_from_source.md (hunk 4 above)
Comment (dkalinowski) [reply docs/build_from_source.md:63]: listed

File: docs/build_from_source.md (hunk 3 above)
Comment (dkalinowski) [reply docs/build_from_source.md:91]: added

File: docs/build_from_source.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+└── thirdparty-licenses
+```
+
+## Building Options
+
+### `BASE_OS`
+
+Select base OS:
+- `ubuntu` for Ubuntu 20.04 (default)
+- `redhat` for RedHat 8.7
```
Comment (dtrawins) [inline docs/build_from_source.md:53]:
```suggestion
- `redhat` for Red Hat UBI 8.7
```

**Review by dtrawins — APPROVED** [review]

**Review by atobiszei — APPROVED** [review]

**Pull request merged**